PcapDotNet/src/PcapDotNet.Core.Test/WiresharkCompareTests.cs

```diff
@@ -390,24 +391,58 @@ private static void CreateRandomIpPayload(Random random, Layer ipLayer, List<ILa
             }
         }
 
-        private static void CreateRandomUdpPayload(Random random, UdpLayer udpLayer, List<ILayer> layers)
-        {
-            if (random.NextBool(20))
-            {
-                // Finish with payload.
-                PayloadLayer payloadLayer = random.NextPayloadLayer(random.Next(100));
-                layers.Add(payloadLayer);
-                return;
-            }
-
-            DnsLayer dnsLayer = random.NextDnsLayer();
-            layers.Add(dnsLayer);
-
-            ushort specialPort = (ushort)(random.NextBool() ? 53 : 5355);
-            if (dnsLayer.IsQuery)
-                udpLayer.DestinationPort = specialPort;
-            else
-                udpLayer.SourcePort = specialPort;
+        private static void CreateRandomUdpPayload(Random random, UdpLayer udpLayer, List<ILayer> layers)
+        {
+            if (random.NextBool(20))
+            {
+                // Finish with payload.
+                PayloadLayer payloadLayer = random.NextPayloadLayer(random.Next(100));
+                layers.Add(payloadLayer);
+                return;
+            }
+
+            switch (random.Next(0, 2))
```

Comment:
I would keep it because there may be more than two possible payloads in the future